vue/components/ui/organisms/listing/listing.vue

```diff
@@ -17,14 +17,30 @@
             <template v-slot:header-before>
                 <slot v-bind:name="'header-before'">
                     <slot name="header-search">
-                        <search
-                            class="search-mobile"
-                            v-bind:variant="'dark'"
-                            v-bind:placeholder="filterText ? filterText : `Search ${name}`"
-                            v-bind:value.sync="filter"
-                            v-bind:loading="loading"
+                        <div
+                            class="search-container"
                             v-if="isMobileWidth()"
-                        />
+                            v-on:mouseover="onSearchMouseOver"
+                            v-on:mouseleave="onSearchMouseLeave"
+                        >
+                            <search
+                                class="search-mobile"
+                                v-bind:variant="'dark'"
+                                v-bind:placeholder="filterText ? filterText : `Search ${name}`"
+                                v-bind:value.sync="filter"
+                                v-bind:loading="loading"
+                            />
+                            <tooltip

```

Comment:
Why not move the `search` inside the `tooltip` using the `slot` rather than controlling its visibility via props?

Reply:
I understood the part of the slot, but I didn't understood the part about the visibility. Could you please explain?

Reply:
The tooltip should take care of that already. I don't know why but your tooltip seems different from this one: https://github.com/ripe-tech/ripe-white/blob/master/vue/components/molecules/tooltip/tooltip.vue

This means you shouldn't need to use any prop for visibility